integration_test/third_party_apps_data/applications/mongodb/debian_ubuntu/install

```diff
@@ -9,6 +9,12 @@ osname="$(lsb_release -si)"
 
 case $osname in 
   Ubuntu)
+    if [ "$oscode" = "lunar" ]; then
+      # Ubuntu 23.04 is not officially supported, so use 'focal'.
+      oscode="focal"
+      wget http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2_amd64.deb
+      sudo dpkg -i libssl1.1_1.1.1f-1ubuntu2_amd64.deb
```

Comment:
This is confusing — I thought only MongoDB 3.6 required libssl1… Does 5.0 require it as well?

Reply:
Both do, yes.